workers/docker-worker/test/integration/container_startup_test.js

```diff
@@ -26,14 +26,14 @@ helper.secrets.mockSuite(suiteName(), ['docker', 'ci-creds'], function(mock, ski
       payload: {
         image: 'taskcluster/test-ubuntu',
         command: [
-          'echo "Hello"',
+          '/usr/bin/no-such-command',
         ],
         maxRunTime: 30,
       },
     });
 
-    assert.equal(result.run.state, 'failed', 'task should be successful');
```

Comment:
huh weird. copy paste maybe?

Reply:
Yeah -- between that and the clever selection of `/usr/bin/echo "hello"` as the not-found command, this one took a _long_ time to figure out..